<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IPSPatcherUIClass</class>
 <widget class="QWidget" name="IPSPatcherUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IPSPatcherUI</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IPS File:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipsPath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="convertBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filePath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>File to patch:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="fileBrowseBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ipsBrowseBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="overwriteCheck">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Overwrite original file with output</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outPath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Output:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outBrowseBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="IPSPatcherUI.qrc"/>
 </resources>
 <connections/>
</ui>
